app/models/school_data_importer.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require "file_download"
+require "csv"
+
+class SchoolDataImporter
+  # TODO: Register and Partner 147: Figure out how exactly we want to handle school updating
+  def run
+    CSV.foreach(schools_data_file.path, headers: true, encoding: "ISO-8859-1:UTF-8") do |row|
+      school = row_to_school(row)
+      school.save!
+    end
+  end
+
+private
+
+  def gias_schools_csv_url
+    date_string = Time.zone.now.strftime("%Y%m%d")
+    "https://ea-edubase-api-prod.azurewebsites.net/edubase/downloads/public/edubasealldata#{date_string}.csv"
```

Comment:
The base part of this definitely needs to go in an env file. They could change their server in future